backstop_data/engine_scripts/puppet/mailings/ab-test-manage.js

```diff
@@ -3,8 +3,8 @@
 module.exports = async (engine, scenario, vp) => {
   await engine.waitFor('table.display', { visible: true });
 
-  // The list grows on each passing run, so we display only 10 rows in the test
+  // The list grows on each passing run, so we display only 9 rows in the test (First being the main row);
   await engine.evaluate(() => {
-    document.querySelectorAll('table.display tr:nth-child(n+10)').forEach(row => row.remove());
+    document.querySelectorAll('table.display tr:nth-child(n+4)').forEach(row => row.remove());
```

Comment:
why was this change necessary at all?

Reply:
Since in the Vanilla CiviCRM there are too few fields and was creating a problem while testing the scenario. 
What is your opinion?

Reply:
I'm not sure i understand.

You changed the comment from "only 10 rows" to "only 9 rows", but then you changed the selector to remove any row past the 4th one.

And I don't understand what you mean by "too few fields". What fields? What sort of problems were you encountering?